llvm2kittel-build/phiTest.c

```diff
@@ -0,0 +1,9 @@
+//
+//  phiTest.c
+//  
+//
+//  Created by Heidy Khlaaf on 24/09/2015.
+//
+//
+
+#include <stdio.h>
```

Comment:
Ah, this was added when I was testing it with Marc. It is not necessary and I will remove this.